src/modules/CreateClassForm/components/CreateClassForm.js

```diff
@@ -49,7 +51,6 @@ function CreateClassForm({ onSubmit }) {
           name="sponsor"
           options={[{ value: '1', label: 'Cardinal Solutions' }]}
           type={Select}
-          placeholder="Select a sponsor"
```

Comment:
Why remove the placeholder?

Reply:
I cannot explain why I did this, but I will undo this.